src/component/Clock.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+type State = {
+  today: Date,
+};
+
+type Props = {
+  name: string,
+};
+
+export class Clock extends React.PureComponent<Props, State> {
+  state: State = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount(): void {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      // eslint-disable-next-line no-console
+      console.info(`${this.state.today.toUTCString().slice(-12, -4)}`);
```

Comment:
Maybe this logic shoud be in `componentDidUpdate`?

Reply:
it doesn't work